main.go

```diff
@@ -0,0 +1,167 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/bitrise-io/appcenter"
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-steputils/tools"
+	"github.com/bitrise-io/go-utils/log"
+)
+
+const statusEnvKey = "APPCENTER_DEPLOY_STATUS"
+
+type config struct {
+	Debug              bool            `env:"debug,required"`
+	AppPath            string          `env:"app_path,file"`
+	AppName            string          `env:"app_name,required"`
+	APIToken           stepconf.Secret `env:"api_token,required"`
+	OwnerName          string          `env:"owner_name,required"`
+	Mandatory          bool            `env:"mandatory,required"`
+	MappingPath        string          `env:"mapping_path"`
+	ReleaseNotes       string          `env:"release_notes"`
+	NotifyTesters      bool            `env:"notify_testers,required"`
+	DistributionGroup  string          `env:"distribution_group"`
+	DistributionStore  string          `env:"distribution_store"`
+	DistributionTester string          `env:"distribution_tester"`
+}
+
+func failf(f string, args ...interface{}) {
+	log.Errorf(f, args...)
+	os.Exit(1)
+}
+
+func main() {
+	var cfg config
+	if err := stepconf.Parse(&cfg); err != nil {
+		failf("Issue with input: %s", err)
+	}
+	stepconf.Print(cfg)
+	fmt.Println()
+
+	app := appcenter.NewClient(string(cfg.APIToken), cfg.Debug).Apps(cfg.OwnerName, cfg.AppName)
+
+	log.Infof("Uploading binary")
+
+	release, err := app.NewRelease(cfg.AppPath)
```

Comment:
AppPath can contain multiple APKs/AABs, we have to choose one. Can you add a test for this?

Reply:
Correction, AppPath contains a single path.